renderer_cocoa/lib/Cocoa.ml

```diff
@@ -0,0 +1,228 @@
+(**
+   Cocoa bindings for OCaml.
+   Original project by Nicolás Ojeda Bär © https://github.com/nojb/ocaml-cocoa
+*)
+
+module T =
+  Ephemeron.K1.Make (struct
+    type t = int
+
+    let equal = (=)
+    let hash = Hashtbl.hash
+  end)
+
+external _NSLog: string -> unit = "ml_NSLog"
+
+let log fmt =
+  Printf.ksprintf _NSLog fmt
+
+type nsApp
+
+type nsWindow
+
+type nsView
+
+module NSApplication = struct
+  external _NSApplication_NSApp: int -> nsApp = "ml_NSApplication_NSApp" "ml_NSApplication_NSApp"
```

Comment:
I think we should think twice about including those bindings. We don't need them, certainly not in this form. They were written with writing apps using them directly. We are writing another abstraction and we don't need this object oriented layer there I think.

Reply:
I agree with you, but on the other hand I want to keep as less as possible on the 'XCode' side. (not that these bindings are not half-assed)

Reply:
How do you want to handle it?

Reply:
well, we can simply remove `NSApplication` and `NSWindow` for now. They are useless in our current context.